CodeMaid/Integration/Commands/CommentFormatCommand.cs

```diff
@@ -27,7 +27,7 @@ internal CommentFormatCommand(CodeMaidPackage package)
             : base(package,
                    new CommandID(PackageGuids.GuidCodeMaidCommandCommentFormat, PackageIds.CmdIDCodeMaidCommentFormat))
         {
-            _undoTransactionHelper = new UndoTransactionHelper(package, "CodeMaid Format Comment");
+            _undoTransactionHelper = new UndoTransactionHelper(package, StringResourceKey.CommentFormatCommand_CommentFormatCommand_CodeMaidFormatComment);
```

Comment:
StringResourceKey is the name of the string resource class that is used by default when tom-englert/ResXResourceManager is automatically generated. It's not perfect, but it's the best localization translation tool I've ever met.

Reply:
Sometimes StringResourceKey is used.. sometimes Resources is used.  I think to avoid confusion Resources should always be used where possible.

Reply:
Using resources requires referencing namespace SteveCadwallader.CodeMaid.Properties, but using Stringresourcekey does not require Stevecadwallader.codemaid, Because the root namespace is it. It would be awkward to add namespaces to all the required code SteveCadwallader.CodeMaid.Properties. However, it is easier to refer to SteveCadwallader.CodeMaid.Properties in WPF.